From bb23c7b57b42248205fb4a298670f7f6e53ae165 Mon Sep 17 00:00:00 2001
From: Even Brenden <evenbrenden@gmail.com>
Date: Thu, 21 May 2020 11:37:56 +0200
Subject: [PATCH 1/1] PulseAudio 13.99

---
 pkgs/servers/pulseaudio/default.nix | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/pkgs/servers/pulseaudio/default.nix b/pkgs/servers/pulseaudio/default.nix
index b9fb104d5f6..1cd00ff4133 100644
--- a/pkgs/servers/pulseaudio/default.nix
+++ b/pkgs/servers/pulseaudio/default.nix
@@ -31,11 +31,11 @@
 
 stdenv.mkDerivation rec {
   name = "${if libOnly then "lib" else ""}pulseaudio-${version}";
-  version = "13.0";
+  version = "13.99.1";
 
   src = fetchurl {
     url = "http://freedesktop.org/software/pulseaudio/releases/pulseaudio-${version}.tar.xz";
-    sha256 = "0mw0ybrqj7hvf8lqs5gjzip464hfnixw453lr0mqzlng3b5266wn";
+    sha256 = "030a7v0khp6w683km81c6vpch1687pvx2gvscnzkjq4f0z6138g6";
   };
 
   outputs = [ "out" "dev" ];
@@ -106,7 +106,8 @@ stdenv.mkDerivation rec {
 
   postInstall = lib.optionalString libOnly ''
     rm -rf $out/{bin,share,etc,lib/{pulse-*,systemd}}
-    sed 's|-lltdl|-L${libtool.lib}/lib -lltdl|' -i $out/lib/pulseaudio/libpulsecore-${version}.la
+    ls $out/lib/pulseaudio/
+    sed 's|-lltdl|-L${libtool.lib}/lib -lltdl|' -i $out/lib/pulseaudio/libpulsecore-13.99.la
   ''
     + ''
     moveToOutput lib/cmake "$dev"
-- 
2.26.2

